x-pack/plugins/observability/public/components/shared/field_value_suggestions/index.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+
+import { useDebounce } from 'react-use';
+import { useValuesList } from '../../../hooks/use_values_list';
+import { IIndexPattern } from '../../../../../../../src/plugins/data/common';
+import { FieldValueSelection } from './field_value_selection';
+
+export interface FieldValueSuggestionsProps {
+  value?: string;
+  label: string;
+  indexPattern: IIndexPattern;
+  sourceField: string;
+  onChange: (val: string) => void;
+}
+
+export function FieldValueSuggestions({
+  sourceField,
+  label,
+  indexPattern,
+  value,
+  onChange: onSelectionChange,
+}: FieldValueSuggestionsProps) {
+  const [query, setQuery] = useState('');
```

Comment:
since it's initialised with '' i think it's unnecessary to specify type, since it will automatically infer that. 

WDYT? Specifying it would be more readable?

Reply:
Actually that makes complete sense since it's a simple type. I wasn't thinking when I wrote this.